locations/spiders/completude.py

```diff
@@ -31,13 +32,15 @@ def parse_location(self, response):
                 "city": city,
                 "postcode": postal,
                 "phone": response.xpath('//ul[@class="list-contacts"]/li[1]/a/span/text()').extract_first(),
-                "name": response.xpath('//select[@name="agency"]/option[2]/text()').extract_first(),
+                "name": response.xpath('//select[@name="agency"]/option[2]/text()').extract_first()
+                or f"Completude Office in {city}",
```

Comment:
Generally we don't create a name of our own. If they don't give a name, then we shouldn't include it in here.